package/upgrade/upgrade_manifests.sh

```diff
@@ -874,7 +874,7 @@ upgrade_harvester() {
   cat >harvester-crd.yaml <<EOF
 spec:
   version: $REPO_HARVESTER_CHART_VERSION
-  timeoutSeconds: 60
```

Comment:
Looks we did not set this value on a fresh cluster, that's why I encounter it on v1.4.0

https://github.com/harvester/harvester-installer/blob/8ff5804d5dd75c87ee35772d0454c7fd02c3cacd/pkg/config/templates/rancherd-10-harvester.yaml#L72